### PR TITLE
apache pinot 1.0 blog and older blogs

### PR DESCRIPTION
**Pull request by hdulay**
# Adding announcement blog for apache pinot 1.0.
- Join Support - Part of the Multi-Stage Query Engine 
- Improved Upserts - Deletion and Compaction Support
- Encode User-Specified Compressed Log Processor (CLP) During Ingestion
- NULL Support
- Pluggable Index Types [Index Service Provider Interface (SPI)]
- Improved Pinot-Spark Integration - Spark3 Compatibility

# Adding older blogs
Geospatial Indexing in Apache Pinot
Apache Pinot™ 0.12: Consumer Record Lag
Apache Pinot™ 0.12: Configurable Time Boundary
Apache Pinot™ 0.11: Deduplication on Real-Time Tables
Apache Pinot™ 0.11: Pausing Real-Time Ingestion
Apache Pinot™ 0.11: Timestamp Indexes
Apache Pinot™ 0.11: Inserts from SQL
Apache Pinot™ 0.11: How do I see my indexes?
GapFill Function For Time-Series Datasets In Pinot

### CODE REVIEW

**Comment by hdulay** [comment]
@Jackie-Jiang, @xiangfu0 , @mayankshriv 

Please review and approve this blog. Thanks

**Review by mayankshriv — DISMISSED** [review]

File: website/blog/2022-08-02-GapFill-Function-For-Time-Series-Datasets-In-Pinot.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
+---
```
Comment (walterddr) [inline website/blog/2022-08-02-GapFill-Function-For-Time-Series-Datasets-In-Pinot.md:1]: are these relevant changes? seem like an old blog post?

Comment (hdulay) [reply website/blog/2022-08-02-GapFill-Function-For-Time-Series-Datasets-In-Pinot.md:1]: piggy backing this to add old blog posts.

**Comment by xiangfu0** [comment]
Any idea on the ci check failure?

File: website/blog/2023-09-19-Annoucing-Apache-Pinot-1-0.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
+---
+title: Geospatial Indexing in Apache Pinot
```
Comment (xiangfu0) [inline website/blog/2023-09-19-Annoucing-Apache-Pinot-1-0.md]: Title should be Announcing Pinot 1.0.0?

**Comment by walterddr** [comment]
The following frontmatter:
{
  "title": "Geospatial Indexing in Apache Pinot",
  "author": "Hubert Dulay",
  "author_title": "Hubert Dulay, Mayank Shrivastava, Neha Pawar",
  "author_url": "",
  "author_image_url": "",
  "description": "Introducing Apache Pinot 1.0 Release",
  "keywords": [
    "Apache Pinot",
    "Apache Pinot 1.0",
    "joins",
    "upsert",
    "clp compression",
    "null support",
    "pluggable index",
    "spark integration"
  ],
  "tags": [
    "Pinot",
    "Data",
    "Analytics",
    "User-Facing Analytics",
    "joins",
    "compression",
    "null support",
    "pluggable index",
    "spark integration"
  ]
}
contains invalid values for field(s): author_url, author_image_url.
 - "author_url" is not allowed to be empty
 - "author_image_url" is not allowed to be empty

**Comment by walterddr** [comment]
Consider only do 1.0 blogs in the PR. We can back fill the rest separately

**Review by xiangfu0 — APPROVED** [review]

**Pull request merged**